recipe/meta.yaml

```diff
@@ -41,16 +41,22 @@ outputs:
 about:
   home: https://www.rust-lang.org
   summary: A safe systems programming language (conda activation scripts)
-  license: BSD
+  license: BSD-3-Clause
   license_family: BSD
   license_file: LICENSE.txt
+  license_url: https://github.com/AnacondaRecipes/rust-activation-feedstock/blob/master/LICENSE.txt
```

Comment:
I really don't like those license_url tags, if there is already a license_file one